app/routes/case-management.route.js

```diff
@@ -0,0 +1,34 @@
+var authorizer = require('../../authorization/etl-authorizer');
+var resolveLocationUuidToId = require('../../location/resolve-location-uuid-to-id');
+var privileges = authorizer.getAllPrivileges();
+var caseManagementData = require('../../dao/case-management/case-management')
+const routes = [
+    {
+        method: 'GET',
+        path: '/etl/case-management',
+        config: {
+            plugins: {
+                'hapiAuthorization': {
+                    role: privileges.canViewPatient
+                }
+            },
+            handler: function (request, reply) {
+                caseManagementData.getCaseManagementData(request).then((result) => {
```

Comment:
we don't want to tight-couple the request to the caseManagementData. Do you want to extract params explicitly from request object, and pass them to the caseManagementData?

Reply:
on it